pkg/v1/resource/chart/mock_test.go

```diff
@@ -85,6 +85,10 @@ func (h *helmMock) InstallFromTarball(path, ns string, options ...helm.InstallOp
 	return nil
 }
 
+func (h *helmMock) PingTiller() error {
+	return nil
+}
```

Comment:
PingTiller was added to helmclient for monitoring. So it needs to be added to the mocks.